r-seurat-scripts-post-install-tests.bats

```diff
@@ -59,10 +59,35 @@
     
     [ "$status" -eq 0 ]
     [ -f  "$normalised_seurat_object" ]
+	}	
+
+
+
+ # run find transfer anchor with normalised_seurat_object as query and reference
+ 
+@test "Find transfer anchors" {
+   #if [ "$use_existing_outputs" = 'true' ] && [ -f "$normalised_seurat_object" ]; then
+     #   skip "$normalised_seurat_object exists and use_existing_outputs is set to 'true'"
+   # fi
+   
+    run seurat-find-transfer-anchor.R $normalised_seurat_object $normalised_seurat_object
```

Comment:
@racoonrocket - a quick look at https://satijalab.org/seurat/v3.1/integration.html suggests to me that this process comes after PCA, so maybe you want to use $pca_seurat_object rather than one up to normalisation?